### PR TITLE
[WIP] Implement Bool.__rtruediv__()

### PR DESCRIPTION
**Pull request by SrishtiSengupta**
Implemented **rtruediv**

### CODE REVIEW

**Comment by cflee** [comment]
@SrishtiSengupta 
Hmm, this seems to include all the commits in #209 as well, not just those that are Bool related, as per the PR title.

If this PR is just meant to implement `rtruediv`, could you just have a2d9614 on this branch? Otherwise, if it's meant to include other Bool methods (I see you have `truediv` implemented), could you just get all those commits onto this branch?

If you need some guidance on doing that, it would be easier to ping someone on Slack or Gitter than to do it here :)

**Comment by phildini** [comment]
This is looking good! A couple of comments, but it's pretty close to mergable. :)

File: python/common/org/python/types/Bool.java [hunk 1]
```diff
@@ -383,7 +383,13 @@ public boolean __setattr_null(java.lang.String name, org.python.Object value) {
...
     )
     public org.python.Object __rtruediv__(org.python.Object other) {
-        throw new org.python.exceptions.NotImplementedError("bool.__rtruediv__() has not been implemented.");
+        try {
+            org.python.types.Int other_val = (org.python.types.Int)other;
+            return new org.python.types.Int(other_val.value == 1 ? 1 : 0).__truediv__(this);
+        } catch (org.python.exceptions.TypeError ae) {
+            throw new org.python.exceptions.TypeError("unsupported operand type(s) for //: 'bool' and '" + other.typeName() + "'");
+        }
+        // throw new org.python.exceptions.NotImplementedError("bool.__rtruediv__() has not been implemented.");
```
Comment (phildini) [inline python/common/org/python/types/Bool.java]: Can you remove this line? We don't want to leave it in if it's no longer necessary.

Comment (SrishtiSengupta) [inline python/common/org/python/types/Bool.java]: @phildini I've removed the unnecessary comments. I guess it can be merged now :)

**Comment by freakboy3742** [comment]
There should be some sort of test for this new method. Adding new behavior without adding a test as well is a red flag.

**Comment by goanpeca** [comment]
@SrishtiSengupta any update on a test for this?

**Comment by SrishtiSengupta** [comment]
@goanpeca Oh dear! I completely lost track of this after my summer of code ended. I'll try to get this done over the next few weeks.

**Comment by cflee** [comment]
Thanks for your contribution! However, this test case is currently passing in master, so I'm going to go ahead and close this PR.

**Pull request closed without merging**